src/main/java/com/example/bookstore/controller/BookController.java

```diff
@@ -35,4 +38,20 @@ public BookDto createBook(@RequestBody CreateBookRequestDto bookDto) {
         return bookService.save(bookDto);
     }
 
+    @PutMapping("/{id}")
+    public BookDto updateBook(@PathVariable Long id,
+                              @RequestBody CreateBookRequestDto createBookRequestDto) {
+        return bookService.updateById(createBookRequestDto, id);
+    }
+
+    @DeleteMapping("/{id}")
+    @ResponseStatus(code = HttpStatus.NO_CONTENT)
```

Comment:
I did because this method doesn't return anything

Reply:
@teract10s I didn't mean that it is wrong to add `@ResponseStatus` here :)

I meant that it would be nice if you've added that to all other endpoints as well

and FYI - when you have `void` method you could leverage `ResponseEntity<Void>` as a return type